src/main/resources/messages/LibertyBundles.properties

```diff
@@ -25,7 +25,7 @@ action.io.openliberty.tools.intellij.actions.ViewUnitTestReport.text=Liberty: Vi
 action.io.openliberty.tools.intellij.actions.ViewUnitTestReport.description=Open the Maven Surefire test report for the corresponding Liberty project
 action.io.openliberty.tools.intellij.actions.ViewTestReport.text=Liberty: View test report
 action.io.openliberty.tools.intellij.actions.ViewTestReport.description=Open the Gradle test report for the corresponding Liberty project
-action.io.openliberty.tools.intellij.actions.ViewEffectivePom.text=Liberty: View effective POM
+action.io.openliberty.tools.intellij.actions.ViewEffectivePom.text=Liberty: View pom.xml
```

Comment:
We should also update the name of the action here and throughout the rest of the code as well. `ViewPom` or `ViewPomXml` would be good.